packages/@okta/vuepress-site/docs/guides/oauth-for-okta-apis/scopes/index.md

```diff
@@ -0,0 +1,45 @@
+| `okta.factors.manage`    | Allows the app to manage all admin operations for org factors (for example, activate, deactive, read)| [Factors API](/docs/reference/api/factors/#factor-operations)|
+| `okta.factors.read`      | Allows the app to read org factors information                          | [Factors API](/docs/reference/api/factors/#factor-operations)|
+| `okta.groups.manage`     | Allows the app to manage groups in your Okta organization               | [Groups API](/docs/reference/api/groups/#getting-started-with-the-groups-api)|
+| `okta.groups.read`       | Allows the app to read information about groups and their members in your Okta organization| [Groups API](/docs/reference/api/groups/#getting-started-with-the-groups-api)|
+| `okta.groups.register`   | Allows the app to create new groups in your Okta organization           | [Add Groups API](/docs/reference/api/groups/#add-group)|
+| `okta.idps.manage`       | Allows the app to create and manage Identity Providers in your Okta organization| [Identity Providers API](/docs/reference/api/idps/#getting-started)|
+| `okta.idps.read`         | Allows the app to read information about Identity Providers in your Okta organization| [Identity Providers API](/docs/reference/api/idps/#getting-started)|
+| `okta.inlineHooks.manage`| Allows the app to create and manage Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
+| `okta.inlineHooks.read` | Allows the app to read information about Inline Hooks in your Okta organization.| [Inline Hooks API](/docs/reference/api/inline-hooks/)|
+| `okta.logs.read`         | Allows the app to read information about System Log entries in your Okta organization| [System Log API](/docs/reference/api/system-log/)|
+| `okta.schemas.manage`    | Allows the app to create and manage Schemas in your Okta organization   | [Schemas API](/docs/reference/api/schemas/#getting-started)|
+| `okta.schemas.read`      | Allows the app to read information about Schemas in your Okta organization| [Schemas API](/docs/reference/api/schemas/#getting-started)|
+| `okta.users.manage`      | Allows an app to manage any user's profile and credential information and to create new users| [Users API](https://developer.okta.com/docs/reference/api/users/#user-operations), [User Lifecycle Operations](/docs/reference/api/users/#lifecycle-operations)|
+| `okta.users.read`        | Allows an app to read any user's profile and credential information      | [Users API](https://developer.okta.com/docs/reference/api/users/#user-operations), [User Lifecycle Operations](/docs/reference/api/users/#lifecycle-operations)|
+| `okta.users.manage.self` | Allows an app to manage the currently signed-in user's profile           |   |
+| `okta.users.read.self`   | Allows an app to read the currently signed-in user's profile and credential information|  |
+
+## Scope Hierarchy 
```

Comment:
I'd position this as "scope naming" instead of Hierarchy. This section was confusing to me.